config/pre-install/v0.19.0/storage-version-migration.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: storage-version-migration-knative-gcp-v0.19.0
+  namespace: cloud-run-events
+  labels:
+    app: "storage-version-migration-knative-gcp"
```

Comment:
Can you change the label to make it match the name?

Reply:
done

Reply:
Do we want to change the naming convention in clusterrole.yaml and serviceaccount.yaml as well? (i.e. adding the v0.19.0 suffix)

Reply:
I believe so. Otherwise, I think we will encounter the same issues as you mentioned for the migration job when we tried to upgrade from 0.17 to 0.18, then from 0.18 to 0.19.

Reply:
I changed all suffix from `v0.19.0` to `v0-19-0`.